docs/source/daml/reference/templates.rst

```diff
@@ -184,3 +184,24 @@ Contract Keys and Maintainers
 
   Because of this, the ``key`` must include the ``maintainer`` ``Party`` or parties (for example, as part of a tuple or record), and the ``maintainer`` must be a signatory.
 - For a full explanation, see :doc:`/daml/reference/contract-keys`.
+
+Interface Instances
+*******************
+
+.. warning::
+  This feature is under active development and not officially supported in
+  production environments.
+
+.. literalinclude:: ../code-snippets-dev/Interfaces.daml
```

Comment:
Our code snippets are still built targeting LF 1.14, so instead of adding an `interface instance` in `../code-snippets/Reference.daml`, we reuse the one from Reference: Interfaces.